src/outfit.ts

```diff
@@ -224,6 +224,13 @@ export class Outfit {
       outfit.modifier = spec.modifier;
     }
 
+    const commonFamiliarEquips = new Map<Familiar, Item>([
```

Comment:
This should go at the end of the equipDefaults. The order of operations in the engine is .create(task) to get equipment required by the task, then it assigns resources, then calls .equipDefaults to fill out the remaining slots.